report_xls_template/__openerp__.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Addons modules by CLEARCORP S.A.
+#    Copyright (C) 2009-TODAY CLEARCORP S.A. (<http://clearcorp.co.cr>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'XLS Reports',
+    'version': '1.0',
+    'category': 'Base',
+    'sequence': 16,
+    'summary': 'XLS format QWeb Reports',
+    'description': """
+Reports in XLS format
+=====================
+Allow you to print reports on xls format.
+
+Main Features
+-------------
+* Uses QWeb templates
+* Allows the creation of worksheets
+* Supports base 64 encoded 24 bits True Color bitmaps""",
+    'author': 'ClearCorp',
+    'website': 'http://clearcorp.co.cr',
+    'complexity': 'normal',
+    'images': [],
+    'depends': ['report'],
```

Comment:
complexity and images are deprecated.
also remove description, so that the README can be used